### DISCUSSION TITLE
「开源之史」系列之七/四：科学共同体的礼物——学术型开源许可的诞生 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#47] [Announcements]
# 「开源之史」系列之七/四：科学共同体的礼物——学术型开源许可的诞生 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/02-04-ip-law-and-license-evolution-os-software-license-occur/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
其实吧, 对比 BSD 和其它许可证, 
其气质, 最大的特征, 可能不是学术, 
而是 雅痞, 也可以认为这是学术气质的别称,
是挎掉的一代最有代表性的气质;

主要执行共同体, 就是那些能拿到终身教授职称, 一生不怕饿死街头的精英们,
对待一切自己不感兴趣的事儿, 
都是一副认真而无聊的表情;

对于开源, 其实从
莱昂氏UNIX源代码分析 (Lion's Commentary on UNIX with Source Code)
https://book.douban.com/subject/1231822//

这部书的形成故事就可以感受到 BSD 们的态度,
反正你们看不懂, 我们开源了也一样...

这其实并不是什么好态度...
不过, 正是 自由的追求, 和开源务实的结构,
可以忠诚的将这种态度记载为许可证,
永久的展示智商的优越...

**Reply by lijiangsheng1** [reply]
精英的傲慢～ 黑客的暗面～
